configs/unet/pspnet_unet_s5-d16_128x128_40k_egohands.py

```diff
@@ -0,0 +1,7 @@
+_base_ = [
+    '../_base_/models/pspnet_unet_s5-d16.py',
+    '../_base_/datasets/egohands.py', '../_base_/default_runtime.py',
+    '../_base_/schedules/schedule_40k.py'
+]
+model = dict(test_cfg=dict(crop_size=(128, 128), stride=(85, 85)))
```

Comment:
I have a little question about the `num_classes` of model, does it require modification?

Reply:
`
     CLASSES = ('background', 'hand1_lt', 'hand1_rt', 'hand2_lt', 'hand2_rt')
`
Should be handled by this code in: mmseg/datasets/egohands.py

Reply:
`num_classes` is the model hyperparameter, which decides the lastest layer output channels of the model.

If egohand dataset has 5 classes, I think should modify the `num_classes=5` of model.

Reply:
How is the best way to do that ? Should I set:
`model[auxiliary_head][num_classes] =5`

Reply:
I have run the current training configuration, and `num_classes` maybe not quite right.

Please confirm that the current model training configuration is correct

Reply:
Please try training with this config configs/fcn/fcn_r50-d8_512x1024_40k_egohands.py . I can confirm that it works for me.

Reply:
OK I'll try it later